### PR TITLE
Update EMS to Xperience, add ProductBoard link

### PR DESCRIPTION
**Pull request by seangwright**
### Motivation

Consistent branding transition from EMS to Xperience and direct link to replacement for Xperience UserVoice site

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

n/a

### CODE REVIEW

**Comment by makma** [comment]
Hi guys, this is a really tricky part. I was thinking about renaming and rebranding those resources a lot. We've got also a discussion with @ondrabus about these exact links as well on Friday. 

In my opinion, there's no official Xperience product yet from a technical point of view - product is still called Kentico (EMS) 12, hotfixes are being released for Kentico 12, we've got all the Nugets and libraries for K.12 as well as documentation is for Kentico 12 only. Since these resources are mainly for technical folks, considering the facts above facts, we've decided it makes sense to stick with the old names, for now, to be coherent (since this change might bring some confusion for some users).

Once we release and have Xperience product we will definitely rename all the mentioned occurrences.

What are your thoughts @seangwright ?

PS: Products suggestions link is applicable here.

**Comment by seangwright** [comment]
That all makes sense to me.

A few things to note:

The summary at the top says `Below is a non-exhaustive list of developer resources related to Kentico Xperience and Kentico Kontent.` Should this include EMS?

And, under `Community` is uses the term Xperience instead of EMS.

With content unrelated to Kentico's GitHub presence (like my blogs), should I continue using the EMS/CMS terminology until K13 comes out and then switch to Xperience?

**Comment by makma** [comment]
> The summary at the top says Below is a non-exhaustive list of developer resources related to Kentico Xperience and Kentico Kontent. Should this include EMS?

Yes, it makes sense to me to include all three products there.

> And, under Community is uses the term Xperience instead of EMS.

Yes, it uses. This is not IMO strictly tied to product and resources and we wanted to make it consistent with the Devnet naming.

> With content unrelated to Kentico's GitHub presence (like my blogs), should I continue using the EMS/CMS terminology until K13 comes out and then switch to Xperience?

Tough question. From my perspective, in the perfect world, a piece of content that is tied with a specific product should use the name of the product, therefore EMS. Moreover, if it's applicable for the Kentico Xperience (aka K13), it should be mentioned or renamed after the official launch. I realize this approach might bring some redundant workload. Nevertheless, I don't have a strong opinion about this.

What hat do you think @ondrabus?

**Comment by makma** [comment]
Despite the mentioned facts, I reckon the suggested changes are valuable. On the other hand, this will be domain Ryan's domain. Are you OK with these naming changes and is it aligned with your vision @OnyxPrime?

**Comment by seangwright** [comment]
Since the primary purpose of this document is to help developers and not to market a product, we should include all the information that is relevant to developers - including an explanation of the Xperience / EMS product relationship or using both product names throughout the document.

For example, we have Kontent and Xperience documentation links in the table of contents but no reference to EMS. 

Developers that only know about EMS and haven't kept up to date with all the product naming changes (Kentico 12 EMS, Kentico 12 MVC, Kentico Raptor, Kentico 2020, Kentico 13, Kentico Xperience 13, Kentico Phoenix 🤦‍♂️🤦‍♂️🤦‍♂️), __need our help__ and this document is our opportunity to orient them.

We should keep that in mind when adding/removing/describing information here.

**Comment by OnyxPrime** [comment]
@seangwright I like the idea of updating these references and share your concerns for developers who might be wondering who moved their "cheese" 😁. What are your thoughts around adding something after the name changes like '(fomerly EMS)' to signify the product name change? This would be temporary until the 2 names become synonymous.

**Comment by seangwright** [comment]
I think doing one `(formerly EMS)` at the top is good, and also we should have a link to the best doc/post/page about the branding change. It might also be worth having a link about the branding change from Kentico Cloud to Kentico Kontent.

There's so many times that I miss the jump from one technology name to another and being able to understand that history is helpful, but instead it gets hidden 😯 by marketing.

File: RESOURCES.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Developer Resources
 
-Below is a non-exhaustive list of developer resources related to Kentico Xperience and Kentico Kontent.
+Below is a non-exhaustive list of developer resources related to [Kentico Kontent](https://kontent.ai/), and [Kentico Xperience](https://xperience.io/) (formerly EMS).
+
+> Kentico EMS [was rebranded](https://xperience.io/discover/blog/2020-05/accelerate-your-digital-transformation) to Kentico Xperience in May, 2020. Any version of the product before version 13 is still known as Kentico EMS.
```
Comment (seangwright) [inline RESOURCES.md:5]: I added this description here so we don't have to repeat "(formerly EMS)" everywhere below. We can remove it in the future when the naming is well understood.

File: RESOURCES.md [hunk 2]
```diff
@@ -34,9 +38,13 @@ Below are links to various places you can find the Kentico developer community.
 
 - [Kentico Xperience DevNet](https://devnet.kentico.com/) - The central resource for the Kentico Xperience developer community
 
-- [Kentico Community Slack](https://kentico-community.slack.com/) - A Slack workspace open to Kentico Gold Partners, invite-only to everyone else. If you'd like to join, contact one of the Kentico developer advocates or MVPs [below on Twitter](#kentico-mvps)
+- [Kentico Community Slack](https://kentico-community.slack.com/) - A Slack workspace open to Kentico Gold Partners, invite-only to everyone else. If you'd like to join, contact one of the Kentico developer advocates or MVPs [below on Twitter](#kentico-mvps).
+
+  Technical discussions about, and developer advice for both Kontent and Experience can be found here.
+
+- [Kentico Kontent MVPs](https://kontent.ai/blog/mvp-announcement) - List of all Kentico Kontent MVPs
```
Comment (seangwright) [inline RESOURCES.md:45]: I need the correct links for both the Kontent MVP page and Xperience MVP page.

File: RESOURCES.md [hunk 3]
```diff
@@ -48,73 +56,81 @@ Below are links to various places you can find the Kentico developer community.
...
-- [Kentico](https://twitter.com/Kentico) - Kentico
+- [Kentico](https://twitter.com/Kentico) - Kentico Company account
 
-- [Kentico Kontent](https://twitter.com/KenticoKontent) - Kentico Kontent
+- [Kentico Kontent](https://twitter.com/KenticoKontent) - Kentico Kontent product
 
-- [Kentico Design](https://twitter.com/KenticoDesign) - Kentico's internal designers
+- [Kentico Xperience](https://twitter.com/KenticoXP) - Kentico Xperience product
 
-- [Michal Kadák (Kentico Xperience)](https://twitter.com/MichalKadak) - Product Director - Kentico Xperience
```
Comment (seangwright) [inline RESOURCES.md:61]: I removed Michal's stuff throughout the doc. Most of it is product updates for EMS v12 and 13 that won't be as relevant once the product launches.

File: RESOURCES.md [hunk 4]
```diff
@@ -48,73 +56,81 @@ Below are links to various places you can find the Kentico developer community.
...
+- [Kentico Design](https://twitter.com/KenticoDesign) - Kentico's internal designers
 
-- [Ondřej Chrastina (Kentico Xperience/Kentico Kontent)](https://twitter.com/ChrastinaOndrej) - Kentico Developer Advocate
+- [Ondřej Chrastina (Kentico Kontent)](https://twitter.com/ChrastinaOndrej) - Kentico Kontent Developer Advocate
```
Comment (seangwright) [inline RESOURCES.md:73]: I updated the devrel product associates. Am I correct that @OnyxPrime is the only Xperience devrel person?

Comment (OnyxPrime) [reply RESOURCES.md:73]: You are correct. I am currently the only Xperience DevRel member

File: RESOURCES.md (hunk 2 above)
Comment (OnyxPrime) [reply RESOURCES.md:45]: Here's the Xperience MVP page https://xperience.io/partnership/mvp-program

**Pull request merged**